packages/features/auth/lib/next-auth-options.ts

```diff
@@ -715,9 +716,11 @@ export const AUTH_OPTIONS: AuthOptions = {
             !existingUserWithEmail.emailVerified &&
             !existingUserWithEmail.username
           ) {
-            await prisma.user.update({
+            await prisma.user.updateMany({
               where: {
                 email: existingUserWithEmail.email,
+                username: null,
+                emailVerified: null,
```

Comment:
Providing more fields to better select the user in question in this if statement.